cadasta_test_suites.py

```diff
@@ -115,16 +115,26 @@
 
 # Get all tests from accounts test classes
 new_registration = unittest.TestLoader().loadTestsFromTestCase(NewRegistration)
-registration_attempt_existing_username = unittest.TestLoader().loadTestsFromTestCase(RegistrationAttemptUsernameNotAvailable)
```

Comment:
I'm feeling a bit uneasy about this whole script. Isn't there a way to discover tests automatically? Every time we add a new test class we have to go here and add it manually. 

At least we can simplify the creation of the test suites a bit:

```python

def create_test_suite(cases):
    unittest.TestSuite([unittest.TestLoader. loadTestsFromTestCase(c) for c in cases])

cadasta_organizations_test_suite = create_test_suite([
    CreateOrganization,
    EditOrganization,
    OrganizationArchive,
    ...
])

```

There's a lot of boilerplate in there that makes it difficult to understand what happens in the script